README.md

```diff
@@ -230,10 +230,10 @@ import { myContainer } from "./inversify.config";
 import { TYPES } from "./types";
 import { Warrior } from "./interfaces";
 
-const ninja = myContainer.get<Warrior>(TYPES.Warrior);
+const warrior = myContainer.get<Warrior>(TYPES.Warrior);
 
-expect(ninja.fight()).eql("cut!"); // true
-expect(ninja.sneak()).eql("hit!"); // true
+expect(warrior.fight()).eql("cut!"); // true
+expect(warrior.sneak()).eql("hit!"); // true
```

Comment:
I think that the word choice here is trying to say that a Ninja implements the Warrior interface and is therefor a warrior. Please close this if you agree